specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4475,6 +4517,125 @@
 		},
 		"description": "Response from a List skillset request. If successful, it includes the full definitions of all skillsets."
 	},
+	"TextExtractionAlgorithm": {
+		"type": "string",
+        "enum": [
+          "printed",
+          "handwritten"
+        ],
+        "x-ms-enum": {
+          "name": "TextExtractionAlgorithm",
+          "modelAsString": false
+        },
+		"description": "A value indicating which algorithm to use. Default is printed."
+	},
+	"TextSplitMode": {
+		"type": "string",
+        "enum": [
+          "pages",
+          "sentences"
+        ],
+        "x-ms-enum": {
+          "name": "TextSplitMode",
+          "modelAsString": false
+        },
+		"description": "A value indicating which split mode to perform. The supported values are 'pages' and 'sentences'."
+	},
+	"VisualFeatures": {
+		"type": "string",
+        "enum": [
+          "Categories",
+          "Tags",
+		  "Description",
+		  "Faces",
+		  "ImageType",
+		  "Color"
+        ],
+        "x-ms-enum": {
+          "name": "VisualFeatures",
+          "modelAsString": false
+        },
+		"description": "The strings indicating what visual feature types to return."
+	},
+	"Details": {
+		"type": "string",
+        "enum": [
+          "Celebrities",
+          "Landmarks"
+        ],
+        "x-ms-enum": {
+          "name": "Details",
+          "modelAsString": false
+        },
+		"description": "A string indicating which domain-specific details to return. The supported values are 'Celebrities' and 'Landmarks'."
+	},
+	"Categories": {
+		"type": "string",
+        "enum": [
+          "Location",
+          "Organization",
+		  "Person"
+        ],
+        "x-ms-enum": {
+          "name": "Categories",
+          "modelAsString": false
+        },
+		"description": "A string indicating which named entity categories to return. The supported values are 'Location', 'Organization', and 'Person'."
+	},
+	"SentimentSkillSupportedLanguageCodes": {
+      "properties": {
+        "name": {
+          "type": "string"
+        }
+      },
+      "description": "The language codes supported by SentimentSkill.",
```

Comment:
The descriptions should be a little more explicit about what "supported" means. Does it mean supported for input?